Sequence diagrams/Register death message.txt

```diff
@@ -0,0 +1,15 @@
+title Register death message
+
+participant EPJ
+
+participant Dødsmelding FHIR API
+
+
+EPJ->Dødsmelding FHIR API:Search for Patient based on Identifier (FNR/DNR)
+EPJ<--Dødsmelding FHIR API:Return Patient
```

Comment:
Updated to say that we return a Bundle, since this is what the FHIR specification wants. We probably just have to put the SearchResponse inside a Bundle and that's it.